resources/views/about-us.blade.php

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <link rel="shortcut icon" type="image/x-icon" href="{{ asset('img/Logo.png') }}" />
+    <link rel="stylesheet" type="text/css" href="css/app.css">
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+    <title>Motus</title>
+</head>
+
+<body>
+    <div id="app">
+        @include('partials.nav-partials')
+        <div id="footer" class="container">
+            <br><br><br>
+            <div class="row text-center text-white text-xs-center text-sm-left text-md-left mt-5">
+                <div class="col-xs-10 offset-5">
+                    <h4 class="mb-4"><span><strong>Meet The Team</strong></span></h4>
+                    <hr>
+                    <ul class="list-unstyled quick-links">
```

Comment:
Do you think its better if we use cards here instead of a list ?
https://getbootstrap.com/docs/4.0/components/card/